source/Mask.cpp

```diff
@@ -434,19 +434,19 @@ const vector<vector<Point>> &Mask::Outlines() const
 
 
 
-Mask Mask::operator*(double scale) const
+Mask Mask::operator*(Point scale) const
 {
 	Mask newMask = *this;
 	for(auto &outline : newMask.outlines)
 		for(Point &p : outline)
 			p *= scale;
-	newMask.radius *= scale;
+	newMask.radius *= scale.Length();
```

Comment:
For the "scale of unity", (1, 1), (no change in size) `scale.Length()` is ~1.41.
This doesn't seem correct.
Given that `radius` is the maximum distance from the centre of the sprite, you're probably going to need to recalculate it with the scaled outline.

Consider a scale factor with a very small x value and a very large y value.
If it is applied to a very wide but very short sprite, it may not change the radius at all.
But, if that sprite were rotated by 90° (same radius, but now very tall and narrow), applying the same scale would result in a very different radius.